src/builtinHelpers.js

```diff
@@ -78,6 +78,11 @@ let section = function section(context, params, bodies, td) {
   let key = params.key.split('.');
   let val = td.get(context, key);
   let body, ctx;
+  Object.keys(params).forEach((key) => {
+    if (key !== 'key') {
+      td.helperContext.set(key, params[key]);
```

Comment:
When does the helperContext get cleared out.  Could these context vars leak into other things?